cmd/juju/application/store/charmadapter_test.go

```diff
@@ -184,7 +182,7 @@ func (s *resolveSuite) expectCharmResolutionCallWithAPIError(curl *charm.URL, ou
 		},
 		Error: err,
 	}}
-	s.charmsAPI.EXPECT().ResolveCharms(gomock.Any()).Return(retVal, nil)
+	s.charmsAPI.EXPECT().ResolveCharms(gomock.Any()).Return(retVal, err)
```

Comment:
This change is incorrect. The error supplied is related to a specific charm, not the call to ResolveCharms.

Reply:
So it is. I don't think I spotted that `WithAPIError` returned nil here originally

This change isn't really necessary, it was just an attempt to clean up a bit of code. Will just revert